src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,385 @@
+
+## Message throughput and rate management
+
+In task-oriented chat, each room is typically low throughput. Scaling comes by running as many rooms as you need at the same time. Ably handles everything from 1:1 conversations and small group discussions to [livestream chats](/docs/guides/chat/build-livestream) with the same reliability and scalability guarantees.
+
+Chat rooms (and Pub/Sub channels) scale horizontally. Ably allows you to have as many rooms as you need running in parallel.
+
+* **Proven at scale:** Ably delivers over 500 million messages per day for customers.
+* **Rate limiting:** Prevent spam and maintain conversation quality with per-user rate limits or global throttling appropriate for your use case.
+
+## Authentication
+
+Authentication is a design decision. 
+
+In some cases, you'll want to tie participants to their identity in your application.
+In others, anyone can join and participate.
+Ably Chat supports both of these scenarios - you don't need to go through the hassle of setting up
+users in Ably and trying to keep them in sync with your application. Every client has a `clientId`.
+If you want to allow random users, just use a random string. If you want to tie it to your application,
+then use the ID of the user or some other well-known identifier. Just make sure you can tell them apart if
+you allow both at the same time, for example by using distinct prefixes.
+
+<Code>
+```javascript
+const jwt = require("jsonwebtoken");
+
+const header = {
+    "typ": "JWT",
+    "alg": "HS256",
+    "kid": "{{ API_KEY_NAME }}"
+}
+const currentTime = Math.round(Date.now() / 1000);
+
+// The clientID could be the users ID in your application,
+// or a random string if you want to make users anonymous.
+//
+// The capabilities here allow the holder of this token to publish and subscribe to messages,
+// typing indicators and room reactions in the chat room called "foo".
+
+const claims = {
+    "iat": currentTime, /* current time in seconds */
+    "exp": currentTime + 14400, /* time of expiration in seconds */
+    "x-ably-capability": "{\"[chat]foo\":[\"publish\", \"subscribe\"]}",
+    "x-ably-clientId": "your-client-id",
+}
+
+ablyJwt = jwt.sign(
+    claims,
+    "{{ API_KEY_SECRET }}",
+    { header: header }
+)
+
+console.log('JWT is: ' + ablyJwt);
+```
+</Code>
+
+How you authenticate is also key. To balance security and experience, you want short-lived tokens that can be easily revoked if a users is misbehaving or needs their permissions changed, but automatically expire after a period of time. This means that if a token is compromised, it will only be valid for a limited time. **In production apps, you should not use API keys for client-side authentication**. You can use them server-side, but as they are long-lived and require explicit revocation, exposure to untrusted users poses a continuing risk.
```

Comment:
```suggestion
How you authenticate is also key. To balance security and experience, you want short-lived tokens that can be easily revoked if a user is misbehaving or needs their permissions changed, but automatically expire after a period of time. This means that if a token is compromised, it will only be valid for a limited time. **In production apps, you should not use API keys for client-side authentication**. You can use them server-side, but as they are long-lived and require explicit revocation, exposure to untrusted users poses a continuing risk.
```